src/client/stylesheets/_gmr-results.scss

```diff
@@ -0,0 +1,95 @@
+@use "govuk-frontend" as *;
+@use "variables" as *;
+
+.gmr {
```

Comment:
all bespoke classes are prefixed with `btms-`
Let's rename this class to `btms-gmr-search-result`